config/locales/api/am_ET.yml

```diff
@@ -14,7 +14,7 @@ am-ET:
       account_locked: "መለያዎ ለሚቀጥሉት %{minutes} ደቂቃዎች ተቆልፏል። እባክዎ ይቆዩ እና እንደገና ይሞክሩ።"
   sms:
     request_otp: "<#>%{otp}የእርስዎ ሲምፕል መተግበሪያ የማረጋገጫ የሚስጥር ቁጥር ነው\n%{app_signature}"
-    appointment_reminders:
+    notifications:
```

Comment:
I'm the one who did this, but I'm not sure we needed to change these translations. They are actually appointment reminders.